src/data/ACTIONS/AST.js

```diff
@@ -258,166 +303,185 @@ export default {
 		cooldown: 30,
 	},
 
-	// -----
-	// Expanded arcana cast from draw slot
-	// -----
-
-	THE_BALANCE_EXPANDED: {
-		id: 4407,
-		name: 'Expanded Balance',
-		icon: 'https://xivapi.com/i/003000/003110.png',
-		cooldown: 30,
+	LADY_OF_CROWNS: {
+		id: 7445,
+		name: 'Lady Of Crowns',
+		icon: 'https://xivapi.com/i/003000/003146.png',
+		cooldown: 5,
 	},
 
-	THE_BOLE_EXPANDED: {
-		id: 4408,
-		name: 'Expanded Bole',
-		icon: 'https://xivapi.com/i/003000/003111.png',
-		cooldown: 30,
+	LORD_OF_CROWNS: {
+		id: 7444,
+		name: 'Lord Of Crowns',
+		icon: 'https://xivapi.com/i/003000/003147.png',
+		cooldown: 5,
 	},
 
-	THE_ARROW_EXPANDED: {
-		id: 4409,
-		name: 'Expanded Arrow',
-		icon: 'https://xivapi.com/i/003000/003112.png',
-		cooldown: 30,
-	},
+	// ----
```

Comment:
All the 4.x stuff is in the `stormblood` branch, no need to keep old data around.